packages/opentelemetry-plugin-document-load/src/documentLoad.ts

```diff
@@ -121,16 +121,14 @@ export class DocumentLoad extends BasePlugin<unknown> {
     const metaElement = [...document.getElementsByTagName('meta')].find(
       e => e.getAttribute('name') === TRACE_PARENT_HEADER
     );
-    const serverContext =
-      parseTraceParent((metaElement && metaElement.content) || '') || undefined;
 
     const entries = this._getEntries();
 
     const rootSpan = this._startSpan(
       AttributeNames.DOCUMENT_LOAD,
       PTN.FETCH_START,
       entries,
-      { parent: serverContext }
+      { parent: parseTraceParent(metaElement?.content ?? '') }
```

Comment:
Looks like optional chaining was finally added in TS3.7... Cool!

Reply:
It only works for node apparently. It breaks the browser test.